lang/en/docs/software-directory/scripting/jupyter-lab/overview.md

```diff
@@ -0,0 +1,26 @@
+# Jupyter Lab
+
+
+JupyterLab [^1] is the next-generation user interface for Project Jupyter [^2] offering all the familiar building blocks of the classic Jupyter Notebook [^3] in a flexible and powerful user interface. JupyterLab will eventually replace the classic Jupyter Notebook.
+
+We include support for **Jupyter Lab** within our platform, using the operating system default interpreter version 2.7.5 for Python.
+
+## Files/Storage Convention
```

Comment:
This should go to a separate section/file, the overview should contain links only. I suggested including it in data earlier. You can create a separate file too.